### PR TITLE
Update tex2Wiki.py

### PR DESCRIPTION
**Pull request by pwang00**

### CODE REVIEW

**Comment by physikerwelt** [comment]
Thank you for your pull request. However there are two problems.
First, the unit tests fail. Please fix that.
Second, your change is too big. Please put the new functionality you added to Azeem/src/tex2Wiki.py
to a new file.
@HowardCohl  do you agree?

**Comment by HowardCohl** [comment]
@physikerwelt 

I agree that he needs to get the unit tests working.

However, for the other part, I don't understand what you want. What do you mean 'new file'?

**Comment by physikerwelt** [comment]
@HowardCohl I think this is now clear from the email discussion?

**Comment by HowardCohl** [comment]
@physikerwelt yes.

**Comment by HowardCohl** [comment]
There are lots of problems with the .mmd files. Just have a look.

**Comment by physikerwelt** [comment]
Please ensure that the travis tests pass. You can click on Details at  continuous-integration/travis-ci/pr to see why they fail

File: Azeem/src/tex2Wiki.py [hunk 1]
```diff
@@ -848,7 +848,7 @@ def KLS_main(n):
                 if eqCounter<endNum:
                     wiki.write("<br /><div id=\"drmf_foot\">\n")
                     if newSec:
-                        newSec=False
+                        newSec=Falseall_funcs = functions
```
Comment (physikerwelt) [inline Azeem/src/tex2Wiki.py]: what does that do?

**Comment by physikerwelt** [comment]
I tried to fix that code for hours now. Unfortunately there are too many changes. And this change reverts a lot of improvements that have been introduced recently. So you need to start over.

**Comment by HowardCohl** [comment]
@pwang00 @pwang00 

This sounds terrible. What on earth happened?

**Pull request closed without merging**